cpp_guidelines/.clang-format

```diff
@@ -0,0 +1,107 @@
+---
+Language: Cpp
+Standard: c++17
+DisableFormat:   false
+AccessModifierOffset: -4
+AlignAfterOpenBracket: Align
+AlignConsecutiveAssignments: true
+AlignConsecutiveBitFields: true
+AlignConsecutiveDeclarations: false
+AlignConsecutiveMacros: true
+AlignEscapedNewlines: Right
+AlignOperands: true
+AlignTrailingComments: true
+AllowAllArgumentsOnNextLine: false
+AllowAllConstructorInitializersOnNextLine: false
+AllowAllParametersOfDeclarationOnNextLine: false
+AllowShortBlocksOnASingleLine: Never
+AllowShortCaseLabelsOnASingleLine: false
+AllowShortEnumsOnASingleLine: false
+AllowShortFunctionsOnASingleLine: None
+AllowShortIfStatementsOnASingleLine: Never
+AllowShortLambdasOnASingleLine: All
+AllowShortLoopsOnASingleLine: false
+AlwaysBreakAfterReturnType: None
+AlwaysBreakBeforeMultilineStrings: false
+AlwaysBreakTemplateDeclarations: Yes
+BinPackArguments: false
+BinPackParameters: false
+BitFieldColonSpacing: Both
```

Comment:
**vote**

|current|remove this option|
|--|---|
0|1

This option is not supported by clang-format-11

```
Error reading /home/widera/workspace/picongpu/.clang-format: Invalid argument
YAML:29:23: error: unknown key 'BitFieldColonSpacing'
BitFieldColonSpacing: Both
```

Reply:
I had the same issue with clang-format 10. I installed 11RC2 and it was still not recognized. Since the option is in the latest documentation, I assume this is an upcoming feature for clang-format 12.

Reply:
current | remove option |
--------|-----------|
 0      |   2

We should postpone this vote until clang 12 arrives.

Reply:
Voting postponed.